src/repos/FlyerRepo.ts

```diff
@@ -233,7 +233,50 @@ const incrementTimesClicked = async (id: string): Promise<Flyer> => {
   return flyer;
 };
 
+/**
+ * Create a new flyer.
+ *
+ * @param {string} categorySlug the slug for this flyer's category
+ * @param {string} endDate the end date for this flyer's event in UTC ISO8601 format
+ * @param {string} flyerURL the URL for this flyer when tapped
+ * @param {string} imageURL the URL representing this flyer's image
+ * @param {string} location the location for this flyer's event
+ * @param {string} organizationID the ID of the organization creating this flyer
+ * @param {string} startDate the start date for this flyer's event in UTC ISO8601 format
+ * @param {string} title the title for this flyer
+ * @returns the newly created Flyer
+ */
+const createFlyer = async (
+  categorySlug: string,
+  endDate: string,
+  flyerURL: string,
+  imageURL: string,
+  location: string,
+  organizationID: string,
+  startDate: string,
+  title: string,
+): Promise<Flyer> => {
+  // Fetch organization given organization ID
+  // This call will fail if the organization cannot be found
+  const organization = await OrganizationModel.findById(new ObjectId(organizationID));
+  const organizationSlug = organization.slug;
+
+  const newFlyer = Object.assign(new Flyer(), {
+    categorySlug,
+    endDate,
+    flyerURL,
+    imageURL,
+    location,
+    organization,
+    organizationSlug,
+    startDate,
+    title,
+  });
```

Comment:
I know you're just following the codebase convention here but at some point this should just be changed to calling `new Flyer(categorySlug, ...)` constructor directly throughout the codebase. No need to change this now but just leaving this here for the future.